package.json

```diff
@@ -5,19 +5,29 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "format": "prettier --write '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "lint": "next lint && prettier --check '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "test": "jest --watch"
```

Comment:
From our project conventions, `test` should be `jest`, and if you want a `--watch` then you can use `test:watch` as a separate run-script.